src/main/java/by/itech/lab/supplier/dto/WarehouseDto.java

```diff
@@ -26,9 +26,6 @@
     @Size(min = 1, max = 10000, message = "Your totalCapacity shouldn't be empty")
     @NotBlank
     private Double totalCapacity;
-    @Size(min = 5, max = 50, message = "Your address should contains at least 5 letters")
-    @NotBlank
     private Address address;
```

Comment:
Do we need to use AddressDto here? It seems like this way we mix models and dto objects, dao level with controller level.

Reply:
Yes, but Address' task is not ready yet. and I can't use AddressDto